utils/tests/searchresultnodate.go

```diff
@@ -0,0 +1,110 @@
+package tests
+
+import (
+	"github.com/jfrog/jfrog-cli-go/artifactory/commands/generic"
+	"github.com/jfrog/jfrog-cli-go/artifactory/spec"
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	clientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+//Used in tests TestArtifactorySearchIncludeDir & 'TestArtifactorySearchProps'.
+//Inorder to compare hard coded json's struction result(e.g. GetSearchPropsStep)
+//witch cannot contain future 'Created' & 'Modified' dates
```

Comment:
I suggest simplifying the above description, which will hopefully make it clearer.
How about:
// Similar to the SearchResult struct, but without the date fields.
// It is used by the tests, which cannot predict the date fields values.

**(Notice also the added white-space after //)**

Reply:
Removed.